eservices-file-generator/src/main/resources/application.conf

```
@@ -0,0 +1,12 @@
+interop-be-eservices-file-generator {
+  storage {
+    kind      = "S3"
+    bucket    = ${DASHBOARD_DATA_BUCKET}
+    filename  = ${DASHBOARD_DATA_FILENAME}
+    asndjson  = "false"
```

this is not required for now, but if we want keep it, we need the possibility to override the configuration
```suggestion
    asndjson  = "false"
    asndjson  = {?IS_NDJSON_FORMAT}
```